pkg/yurthub/yurtcoordinator/resources/resources.go

```diff
@@ -49,6 +50,7 @@ func InitPoolScopeResourcesManger(client kubernetes.Interface, factory informers
 		validPoolScopedResources: make(map[string]*verifiablePoolScopeResource),
 	}
 	configmapInformer := factory.Core().V1().ConfigMaps().Informer()
+	configmapInformer.SetTransform(putil.TransformStripManagedFields())
```

Comment:
It's not needed to set transform here, because the same configmapInformer is used as `cacheAgent`.

Reply:
OK